examples/specialty_plots/ishikawa.py

```diff
@@ -36,108 +36,108 @@ def drawSpines(data:dict,ax:plt.Axes,parentSpine:None,alpha_ps:float=60.0,alpha_
         Relative X-position of primary spines, a list of float is accepted. The default is [np.nan].
     spine_color : str, optional
         Spine color. The default is "black".
-    recLevel : int, optional
+    reclevel : int, optional
         Current recursion level. The default is 1.
-    MAX_RECURSION_LEVEL : int, optional
+    max_recursion_level : int, optional
         Maximum recursion level set. The default is 4.
-    DEBUG : bool, optional
+    debug : bool, optional
```

Comment:
Suggest use the logging module instead of manual debugging.

Reply:
using the logging you can also emit to the logger, but it the log level is set higher it will be automatically dropped so you do not need to thread this parameter through.